ansible/group_vars/all

```diff
@@ -48,3 +48,40 @@ db:
   authkeys:
   - guest
   - whisk.system
+
+# The default value for catalog_source is empty, meaning that
+# openwhisk will not install the catalog from the catalog repositories.
+# If catalog_source is not set to "catalog-repos", the other variables
+# like namespace, api_host, auth_key_file and catalog_repos will not
+# be effective.
+catalog_source: ""
+
+# The default name space is /whisk.system. The namespace must begin with a slash "/".
+namespace: "/whisk.system"
```

Comment:
this is the namespace for installing the catalog? if so, perhaps it should be called catalog_namespace?